models/uniswap/uniswap_schema.yml

```diff
@@ -80,4 +85,39 @@ models:
         description: "address of the trace within the call graph forest"
       - &evt_index
         name: evt_index
-        description: "the position of this event log within this transaction"
\ No newline at end of file
+        description: "the position of this event log within this transaction"
+
+  - name: uniswap_liquidity_manager_pools
+    meta:
+      blockchain: optimism
+      project: uniswap
+      contributors: msilb7
+    config:
+      tags: ['optimism','uniswap','pools','liquidity managers']
+    description: >
+        A table containing Uniswap mappings for known liquidity manager pools created.
+    columns:
+      - &blockchain_lm
```

Comment:
maybe i'm not looking correctly, but these columns don't seem to align with the model itself

Reply:
added the two missing columns in, so they should be good